examples/opensora_hpcai/scripts/train.py

```diff
@@ -368,7 +389,11 @@ def main(args):
         ema=ema,
     )
 
-    model = Model(net_with_grads)
+    if args.global_bf16:
+        model = Model(net_with_grads, amp_level="O0")
```

Comment:
Why is `amp_level` set to `O0` when the training precision is BF16?

Reply:
because only layers in fp32 will be converted bf16 when using `allow_mixed_precision_bf16` to enable operations compute in bf16